package.json

```diff
@@ -32,19 +32,23 @@
     "url": "https://github.com/CrowdStrike/eslint-config-crowdstrike/issues"
   },
   "homepage": "https://github.com/CrowdStrike/eslint-config-crowdstrike#readme",
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
+  },
   "devDependencies": {
     "@crowdstrike/commitlint": "^7.0.0",
-    "eslint": "^8.0.0",
+    "@eslint/js": "^9.7.0",
+    "@eslint/eslintrc": "^3.1.0",
```

Comment:
Do we need this if we removed the .eslintrc.js file?

Reply:
We need it to consume `eslint-config-crowdstrike-node` for now since its still on `<9.x.x`, once its converted to `v9` and outputs a `eslint.config.mjs` of its own this package can consume, we can get rid of this package